antora-ui-camel/src/css/header.css

```diff
@@ -35,6 +35,7 @@ body {
   .navbar-end > .navbar-item,
   .navbar-end .navbar-link {
     color: var(--navbar-font-color);
+    line-height: 55px;
```

Comment:
We prefer not to use px but rather rem as the font size can be changed by the user using the browsers zoom in-out feature.